src/js/utils/underscore.js

```diff
@@ -60,7 +60,7 @@ var _ = function (obj) {
 // The cornerstone, an `each` implementation, aka `forEach`.
 // Handles objects with the built-in `forEach`, arrays, and raw objects.
 // Delegates to **ECMAScript 5**'s native `forEach` if available.
-var each = _.each = _.forEach = function (obj, iterator, context) {
+_.each = _.forEach = function (obj, iterator, context) {
```

Comment:
Export each member as follows. Remove `_` and the default export.
````js
export const each = function() { ... };
export const forEach = each;
```

Reply:
What about `isNaN` and `isFinite`?